models/balancer/arbitrum/balancer_v2_arbitrum_liquidity.sql

```diff
@@ -122,14 +129,14 @@ zipped_balance_changes AS (
             day,
             pool_id,
             token,
-            SUM(COALESCE(amount, 0)) AS amount
+            SUM(COALESCE(amount, CAST(0 as int256))) AS amount
```

Comment:
this'll be my last comment on it, but reminder on the data type approach here as noted above (will apply universally)